tar/bsdtar.c

```diff
@@ -963,7 +963,7 @@ main(int argc, char **argv)
 	network_bwlimit(bsdtar->bwlimit_rate_down, bsdtar->bwlimit_rate_up);
 
 	/* Perform the requested operation. */
-	switch(bsdtar->mode) {
+	switch (bsdtar->mode) {
```

Comment:
The libarchive committer guide specifically states: "Do not make unnecessary
changes to existing whitespace, capitalization, or spelling."

Do you think this counts as "necessary" in freebsd-land?  I wouldn't have
thought that it qualified.